src/notes/FluxNotesEditor.jsx

```diff
@@ -1913,18 +1937,20 @@ class FluxNotesEditor extends React.Component {
                         setOpenedPortal={this.setOpenedPortal}
                         state={this.state.state}
                     />
-                    <ContextPortal
-                        capture={/@([\w]*)/}
-                        callback={callback}
-                        contextManager={this.contextManager}
-                        contexts={this.state.portalOptions}
-                        getPosition={this.getTextCursorPosition}
-                        onChange={this.onChange}
-                        openedPortal={this.state.openedPortal}
-                        onSelected={this.onPortalSelection}
-                        state={this.state.state}
-                        trigger={"@"}
-                    />
+                    {CompletionComponent &&
```

Comment:
Why do we need to expose the completion component to FNE at all? Why can't we just render the completion component within the render method of CompletionPortal?